Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM docker.elastic.co/elasticsearch/elasticsearch:6.4.3
+RUN bin/elasticsearch-plugin install --batch ingest-attachment
```

Comment:
Could we have this file in `docker/elasticsearch` to avoid confusion that it might be for the Wordpress base?